docs/source/use_cases/index.rst

```diff
@@ -1,25 +1,96 @@
+- Possibility to create a light client tracing only related transactions It is possible to make a lightweight client storing only related transactions.   
+- Clear Transaction/Query separation with embedded permissions and role-based access control 
+- Native support for mobile client 
+- Fast and robust  asynchronous consensus 
+- Possibility to tune Iroha to your needs 
+- Narrowed attack space due to the restriction of smart contracts to secure core commands
+- Possibility to implement middleware software for your application   
+
+
+Iroha has an embedded role-based control system with rich permissions set.
+Each command or query in Iroha is verified according to the defined business rules.   
+Permissions in Iroha allow restricting data usage only to a limited set of users reducing the probability of privacy leakage. 
+Moreover, Iroha users can store either encrypted data or even only hash for the data integrity.  
+
+Each asset in Iroha has a specific verified creator. 
+The asset creation and distribution process are totally transparent for the verifiers. 
+We simplified asset creation to just only one command, allowing to tokenize physical items and transfer ownership to other Iroha users. 
+ 
+
+Iroha uses SWIG to connect interfaces written C++ with a variety of high-level programming languages. 
+We flatten the learning curve for developing applications on top of Iroha.  The developer must only learn to use basic Iroha commands to develop a decentralized application with Iroha platform. 
+We hide all blockchain abstractions to enable developer concentrate on creating applications as in typical client-server, without bothering much about blockchain specifics.   
+
+
+Iroha comes as multi-domain blockchain system. Each domain in the system serves as a different subchain.   
+This approach allows creating private or public applications with the possibility to securely interact between different domains.  
+
+
 Use case scenarios
 ==================
+We list some number of abstract use cases and specific advantages that Iroha can introduce to this application. We hope that the applications and use cases will inspire developers and creators to further innovation with Hyperledger Iroha.  
 
-By reviewing application areas together, we can understand what Iroha is capable of providing.
-Use cases here depict the mainstream scenarios of blockchain frameworks.
-It can also be a starting point for your imagination to come up with new and more interesting ways where Iroha can be used.
 
-(In this section we want to see something like https://github.com/takemiyamakoto/hyperledgerwp/tree/master/RelevantUseCases referring to Iroha. The idea is to describe as much as we can so that people can be inspired to use Iroha and can also use their imagination to come up with interesting application. This can be expanded later with good suggestions on hackatons and real-life examples in such use cases.)
+Certificates in Education, Healthcare 
```

Comment:
I expected to see here how a medical institution can store something in Iroha, and a story about people trying to verify here their diplomas or whatever. Without scenarios with a bit of Iroha design part — it is not a section which was planned to be here before (use cases)